jobs.groovy

```diff
@@ -427,6 +438,33 @@ def build_steps(job) {
 }
 
 
+/*
+    Closure that configures the current project to notify slack on failure.
+    The specified channel will be notified.
+*/
+Closure notify_slack(String channel) {
+    return {
+        // I'm not sure how to put these magic strings in a variable
+        // or avoid them altogether.
+        it / 'properties' << 'jenkins.plugins.slack.SlackNotifier_-SlackJobProperty'(plugin: 'slack@1.8.1') {
+            room(values.notify_slack)
+            startNotification(false)
+            notifySuccess(false)
+            notifyAborted(false)
+            notifyNotBuilt(true)
+            notifyUnstable(true)
+            notifyFailure(true)
+            notifyBackToNormal(false)
+            notifyRepeatedFailure(true)
+            includeTestSummary(true)
+            showCommitList(true)
+            includeCustomMessage(false)
+        }
+        it / 'publishers' << 'jenkins.plugins.slack.SlackNotifier'(plugin: 'slack@1.8.1')
```

Comment:
"We couldn't possibly use Haskell, no one would understand it." :(

(no change required)